src-electron/generator/helper-endpointconfig.js

```diff
@@ -617,7 +618,7 @@ async function collectAttributes(endpointTypes, options) {
         // nonempty default value to use long defaults.
         // external strings and zero-length default values.
         if (
-          defaultSize > 2 ||
+          defaultSize > spaceForDefaultValue ||
```

Comment:
@tehampson The documentation in the code above no longer makes sense with this change....